src/bank.rs

```diff
@@ -64,11 +64,20 @@ pub enum BankError {
 }
 
 pub type Result<T> = result::Result<T, BankError>;
+/// An Account with user_data that is stored on chain
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct Account {
+    /// tokens in the account
+    pub tokens: i64,
+    /// user data
+    /// A transaction can write to its user_data
+    pub user_data: Vec<u8>,
```

Comment:
If we can think of nothing more precise, can you call this `userdata` with no underscore? It'll make the code look nicer when you need to qualify it (i.e. `alice_userdata`).